src/lib/rnp.c

```diff
@@ -705,77 +702,96 @@ ptimestr(char *dest, size_t size, time_t t)
 
 /* format a JSON object */
 static void
-format_json_key(FILE *fp, mj_t *obj, const int psigs)
+format_json_key(FILE *fp, json_object *obj, const int psigs)
 {
 	int64_t	 birthtime;
 	int64_t	 duration;
 	time_t	 now;
 	char	 tbuf[32];
-	char	*s;
-	mj_t	*sub;
-	int	 i;
 
 	if (pgp_get_debug_level(__FILE__)) {
-		mj_asprint(&s, obj, MJ_HUMAN);
-		(void) fprintf(stderr, "formatobj: json is '%s'\n", s);
-		free(s);
+		(void) fprintf(stderr, "formatobj: json is '%s'\n", json_object_to_json_string(obj));
 	}
+#if 0 //?
```

Comment:
Good point, i missed to clean this up, and there is a potential issue here:
we handle the scenario when an userid is revoked (see line 767), but this code-part supposed to check whether the whole cert is revoked, however so far i can't find any evidence that this information can get into the generated .json (see the pgp_sprint_mj() function in the lib/packet-print.c file, which might be a flaw already in the original json generation, but i might missed something here, what do you think?